testdata/google/events/cloud/pubsub/v1/MessagePublishedData-simple.json

```diff
@@ -0,0 +1,13 @@
+{
+  "data": {
+    "data": "SGVsbG8gV29ybGQh",
+    "message_id": "3103425958877813",
+    "publish_time": "2021-10-01T20:37:27.386Z"
+  },
```

Comment:
Put just this data in the file. 

I don't think the capitalization is correct (should be lowerCamelCase, not snake_case).

See https://github.com/googleapis/google-cloudevents/blob/main/testdata/google/events/cloud/pubsub/v1/MessagePublishedData-text.json

Reply:
Updated as suggested.